rest/config.go

```diff
@@ -88,7 +89,14 @@ func (dc *DbConfig) MakeBucketSpec() base.BucketSpec {
 	tlsPort := 11207
 
 	if bc.Server != nil {
-		server = *bc.Server
+		// walrus:data represents in memory data store, convert this to rosmar.InMemoryURL
+		// but do not match walrus:///path/to/db, which is incompatible with rosmar URLs
```

Comment:
I left a more detailed comment in Slack about this, but `walrus:data` is a persisted Walrus bucket, with a relative path to `data` (and only if that directory exists... and then it's not persisted 🙃)